modular_skyrat/modules/goofsec/code/clothing/security_eyes.dm

```diff
@@ -0,0 +1,23 @@
+/**
+ * SECURITY EYES OVERRIDES
+ */
+
+/**
+ * Standard security hud
+ */
+/obj/item/clothing/glasses/hud/security/sunglasses
```

Comment:
do we not have prescription sec huds? less a review comment more just a curiosity

Reply:
Not sure tbh.